lang/java/reef-applications/reef-vortex/src/main/java/org/apache/reef/vortex/evaluator/VortexWorker.java

```diff
@@ -109,7 +116,7 @@ public void run() {
 
                       try {
                         // Command Executor: Execute the command
-                        final Serializable result = taskletExecutionRequest.execute();
+                        final Object result = taskletExecutionRequest.execute();
```

Comment:
I think it's difficult to infer the type of the result in this class. But, we can hide `Object` at least, by just passing it to the constructor argument of `TaskletResultReport`. I think that's the best effort we can make for now.